### PR TITLE
Update/UI

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Updated the flow of logic for user and also some UI upgrades

Changes:
1. New User by default now sees Login page instead of Signup
2. Addition of fixed footer to bottom of extension

### CODE REVIEW

File: src/pages/signUp/SignUp.jsx [hunk 1]
```diff
@@ -118,19 +118,22 @@ export const SignUp = () => {
...
+          <span
+            onClick={() => setLoginPage(true)}
+            className="txt-bold txt-underline pointer"
+          >
             Log In!
           </span>
         </span>
       </div>
-      <div className="flex-center">
+      {/* <div className="flex-center">
```
Comment (shivampandey0) [inline src/pages/signUp/SignUp.jsx]: remove commented code if not useful

**Review by shivampandey0 — APPROVED** [review]
Good work with the app flow.

Good to merge

**Review by prachi-sahani — COMMENTED** [review]
Overall code looks fine
Clean UI
Good to merge

File: src/pages/signUp/SignUp.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/pages/signUp/SignUp.jsx]: Done

**Pull request merged**